stix_shifter_utils/stix_translation/src/json_to_stix/json_to_stix.py

```diff
@@ -22,6 +20,13 @@ def translate_results(self, data_source, data):
         except Exception:
             raise LoadJsonResultsException()
 
+        try:
+            json_to_stix_translator = importlib.import_module('stix_shifter_modules.%s.stix_translation.json_to_stix_translator' % self.module_name)
+            print('>>>>>>>>> IMPORT FROM LOCAL json_to_stix_translator!!!!!! ')
+        except Exception as ex:
+            # fall back to default
+            json_to_stix_translator = importlib.import_module('stix_shifter_utils.stix_translation.src.json_to_stix.json_to_stix_translator')
+            print('>>>>>>>>> IMPORT FROM DEFAULT json_to_stix_translator!!!!!! ')
```

Comment:
can you use `from stix_shifter_utils.utils import logger` and pring a logger.info